### ISSUE TITLE
Remove or allow to disable url validation

### ISSUE THREAD

**Issue by ARud12** [opensearch-project/OpenSearch-Dashboards#1740]
How can I remove the URL validation process for the branding icons as well as for the pdf reporting plugin without modifying the source code of OpenSearch Dashboards?

**Comment by kavilla** [comment]
[Triage]: @ARud12, thanks for opening.

There is currently not a way to do this but we will take this as a feature request.

**Comment by ARud12** [comment]
Thank you. Will this feature request include both URL validation for the branding icons as well as the pdf reporting plugin?

**Comment by joshuarrrr** [comment]
I think the best approach to address this use case is an optional config value that allows admins to disable the branding validation checks.

**Comment by ARud12** [comment]
Yes that will be ideal. What is the estimated waiting time for this feature to be incorporated?
